ArduPlane/Log.cpp

```diff
@@ -491,11 +493,12 @@ const struct LogStructure Plane::log_structure[] = {
 // @Field: ArspA:  target airspeed accel
 // @Field: Alt:  target alt
 // @Field: AltA: target alt accel
-// @Field: AltF: target alt frame
+// @Field: AltF: target alt frame (MAVLink)
 // @Field: Hdg:  target heading
 // @Field: HdgA: target heading lim
+// @Field: AltL: target alt frame (Location)
```

Comment:
I don't see the benefit here, were logging the same information twice?

Reply:
I want to know what came in via MAVLink _and_ what the result was (AltFrame)